src/components/TheNavigation.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="navigation-container">
+    <button
+      class="btn-back btn btn-nav"
+      :disabled="isBackBtnDisabled"
+      @click="goBack"
+    >
+      <font-awesome-icon
+        icon="fa-solid fa-angles-left"
+        size="xl"
+        class="icon-left"
+      />
+      Back
+    </button>
+    <button
+      class="btn-forward btn btn-nav"
+      :disabled="isForwardBtnDisabled"
+      @click="goForward"
+    >
+      Forward
+      <font-awesome-icon
+        icon="fa-solid fa-angles-right"
+        class="icon-right"
+        size="xl"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    lastState: {
+      type: Array,
+    },
+
+    id: {
+      type: Number,
+      default: 0,
+    },
+  },
+  data() {
+    return {
+      index: 0,
+      isBackButtonClicked: false,
+      isForwardButtonClicked: false,
+      isBackBtnDisabled: true,
+      isForwardBtnDisabled: true,
+    };
+  },
+  methods: {
+    goBack() {
+      this.isBackButtonClicked = true;
+      this.isForwardButtonClicked = false;
+
+      this.lastState.forEach((state) => {
+        state.isActive = false;
+      });
+      this.index--;
+      this.lastState[this.index].isActive = true;
```

Comment:
Našel sem bug-a, ker prej je bil vezan index na id v array objectu, ampak tam niso id lepo po vrsti, zato sem sedaj rajši implementiral index vezan na klikano kartico. Uporabnik npr. klikne tri kartice, index 3 je najvišji. Z vsakim klikom na kartico(da se kartica odpre) se pusha ta object v array. Če klinem Go back btn, se za vsak klik na ta gumb  index zniža za eno vrednost in zato se  tudi v array[index] , index zniža in odpre kartico vezano na ta index. Obratno za Go forward.

Reply:
to mi je mal confusing... če ti pushaš nekej v array, se da na koncu.. sepravi imaš nek vrsti red in lahko back and forward "potuješ" ane?

če pa nujno rabiš nek property za vrstni red, bi pa jaz dodal `order` in tko pol točno vedu kaj in kako :D 

kot sem napisal spodaj.. te popravke lahko pol daš v drug PR 👍